network-api/networkapi/templates/donate/fragments/formassembly_body.html

```diff
@@ -45,26 +52,14 @@
 
     // Match up locale codes with the locale dropdown entries
                 const localeMap = {
-                    'zh-CN': 'tfa_218',
-                    'zh-TW': 'tfa_218',
-                    'cs': 'tfa_219',
-                    'da': 'tfa_220',
                     'nl': 'tfa_221',
-                    'en-CA': 'tfa_222',
-                    'en-GB': 'tfa_222',
-                    'en-US': 'tfa_222',
-                    'en-ZA': 'tfa_222',
+                    'fy-NL': 'tfa_221',
+                    'en': 'tfa_222',
                     'fr': 'tfa_223',
                     'de': 'tfa_224',
-                    'it': 'tfa_225',
-                    'ja': 'tfa_226',
                     'pl': 'tfa_228',
                     'pt-BR': 'tfa_229',
-                    'ru': 'tfa_230',
-                    'es-MX': 'tfa_231',
-                    'es-AR': 'tfa_231',
-                    'es-ES': 'tfa_231',
-                    'es-CL': 'tfa_231'
+                    'es': 'tfa_231',
                 }
```

Comment:
Hi @mmmavis! [As mentioned here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11306#discussion_r1372527449), I have worked with stakeholders to update this list to language codes that are supported on the foundation site. 

This update was done on FormAssembly, so we wont have to worry about updating this after a copy/paste. It will already be included!

Reply:
Thanks for the note!